pymedphys/tests/dicom/rtplan/test_leaf_jaw_pos.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 Simon Biggs
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pymedphys._dicom import create, rtplan
+
+
+def test_get_leaf_jaw_position_of_type():
+
+    rt_beam_limiting_device_type = "ASYMY"
+    leaf_jaw_positions = [0, 0]
+
+    expected_positions = [leaf_jaw_positions] * 4
```

Comment:
so this means expected_positions = [[0, 0], [0, 0], [0, 0], [0, 0] ] ?
or did you mean expected_positions = [0] * 4
For a single leaf pair Jaw (e.g. Y or ASYMY), there are two positions, e.g. [ -11, 10 ]
If you intend to have the wrong number of values set (as part of your test), then make that clear in the variable naming or at least comment.
Something isn't right here, I'm just not sure what your intent was.  But I think it's the content of the expected_positions.

Reply:
> so this means expected_positions = [[0, 0], [0, 0], [0, 0], [0, 0] ] ?

Yup. Does https://github.com/pymedphys/pymedphys/pull/1048#discussion_r487467296 shed some light on this?

Reply:
Because I added in 3 extra control points:

https://github.com/pymedphys/pymedphys/blob/754eff8c49d1bfbaad4b1062d1a2f2b902e68a00/pymedphys/tests/dicom/rtplan/test_leaf_jaw_pos.py#L41-L43

I expected there to be four control points.